### ISSUE TITLE
github:github/fetch modifies self

### ISSUE THREAD

**Issue by matthewbauer** [jspm/registry#517]
I'm having some issues using the fetch package. The file https://github.com/github/fetch/blob/master/fetch.js seems to modify the 'self' variable which is supposed to correspond to window. In some environments, self is not going to be set. Can it be declared on the global scope?

**Comment by Munter** [comment]
If would be better to do an iffe like this

```js
(function (self) {
  ...
})(this);
```

**Comment by guybedford** [comment]
Yeah I don't think we should be altering the global environment beyond the usual.

Note that you could use the `globals` configuration to set the `self` value to Window via something like:

```javascript
System.config({
  meta: {
    'github:github/fetch@1.2.3/fetch.js': {
       globals: { self: 'window' }
    }
  }
});
```

Where `window.js` is a module containing:

```javascript
module.exports = global;
```

Or something like that.
